### ISSUE TITLE
Tabs: PersistenceException: Unable to commit changes to session

### ISSUE THREAD

**Issue by Kiwinka** [adobe/aem-core-wcm-components#1869]
## Bug Report

TLDR;
when doing operation with tabs I have this error: 
![image](https://user-images.githubusercontent.com/27009823/136819769-1ca8868e-f35b-4658-93ea-ec8dab461414.png)
org.apache.sling.api.resource.PersistenceException: Unable to commit changes to session

**Lore**
I proxied Tabs component, followed the documentation, created the _cq_editConfig.xml file and also added cq:isContainer="true" property. 
Then I created a panel component, which is resourceSuperType of responsivegrid, to represent one panel in the Tabs container.
In the policies of the component I defined that only Panel can be used in Tabs.

**Current Behavior**

When configurating the Tabs component on a page, after adding new panels, or deleting/moving them, I can't save the actions and I'm stopped by the error 'PersistenceException: Unable to commit changes to session'. This does not happen every time, but I couldn't figure out the pattern. It happens when I configure panels through the configuration window. When adding panel with the '+' on the parent component, it does work and they are added, but when I add name to the panel in the Tabs configuration, I can't save the changes.

**Expected behavior**
I expect to be able to edit the Tabs configuration.

**Environment**
- AEM version: AEM 6.5
- Core Components version: 2.17.12
- Tabs version: v1
- JRE version: Java(TM) SE Runtime Environment (build 1.8.0_202-b08)

**Additional info**
[error log.txt](https://github.com/adobe/aem-core-wcm-components/files/7323848/error.log.txt)

I'm attaching the export from error log (only a part of it)

**Comment by Abhishek-F-Secure** [comment]
Hi @ky940819 was this issue resolved somewhere? I am getting this issue now
